zerver/views/documentation.py

```diff
@@ -152,20 +153,24 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
             documentation_article.article_path
         ):
             # Absolute path case
-            article_path = documentation_article.article_path
+            article_full_path = documentation_article.article_path
+        elif self.help_view:
+            article_full_path = (
+                os.path.join(settings.DEPLOY_ROOT, "help") + documentation_article.article_path
+            )
         elif documentation_article.article_path.startswith("/"):
             # Hack: `context["article"] has a leading `/`, so we use + to add directories.
-            article_path = (
+            article_full_path = (
                 os.path.join(settings.DEPLOY_ROOT, "templates") + documentation_article.article_path
             )
         else:
-            article_path = os.path.join(
+            article_full_path = os.path.join(
                 settings.DEPLOY_ROOT, "templates", documentation_article.article_path
             )
 
         # For disabling the "Back to home" on the homepage
         context["not_index_page"] = not context["article"].endswith("/index.md")
-        if self.path_template == "/zerver/help/%s.md":
+        if self.help_view:
```

Comment:
I'm definitely glad we're no longer doing this check via the `path_template` parameter.

Reply:
Yeah, I went ahead an changed that for the policy documentation as well in a prep commit. It's much easier to read this way.